app/(tabs)/group/settings.tsx

```diff
@@ -169,49 +249,97 @@ export default function GroupSettings() {
             onChangeText={(text) => setAmountValue(Number(text))}
           ></TextInput>
         </CustomModal>
+
+        <CustomModal
+          height={300}
+          title="Invite Member"
+          isVisible={inviteModalVisibility}
+          setIsVisible={setInviteModalVisibility}
+          createCallback={inviteMember}
+        >
+          <Text style={[styles.text, { fontSize: 20, marginTop: 10 }]}>
+            Name
+          </Text>
+          <TextInput
+            style={globalStyles.inputField}
+            placeholder="Enter user and hash: User#123"
+            value={newMemberName}
+            onChangeText={(text) => setNewMemberName(text)}
+          />
+        </CustomModal>
+
         <Collapsible title="Members" style={{ marginTop: 6 }}>
           {members.map((member, index) => (
-            <SettingsMember key={index} name={member} />
+            <SettingsMember
+              key={index}
+              name={member}
+              onRemove={() => promptRemoveMember(index)}
+            />
           ))}
           <View
             style={[styles.row, { justifyContent: "center", marginBottom: 8 }]}
           >
-            <UniversalIcon
-              source={IconSource.FontAwesome6}
-              name="circle-plus"
-              size={24}
-            />
             <TouchableOpacity
-              onPress={() => setMembers([...members, "New Member"])}
+              style={styles.row}
+              onPress={() => setInviteModalVisibility(true)}
             >
+              <UniversalIcon
+                source={IconSource.FontAwesome6}
+                name="circle-plus"
+                size={24}
+              />
               <Text style={[styles.text, styles.buttonText]}>
                 Invite Member
               </Text>
             </TouchableOpacity>
           </View>
         </Collapsible>
 
+        <DeleteConfirmationModal
```

Comment:
Vi kunne nok godt have gjort Create/Delete til en prop af CustomModal men ja, if it works then I am happy